### ISSUE TITLE
Plugin does not play nice with freeice.

### ISSUE THREAD

**Issue by Neograph734** [rtc-io/rtc-plugin-temasys#3]
When using the RTC plugin like shown in the examples:
```javascript
var freeice = require('freeice');

call = RTC({
  constraints: {
    audio: true,
    video: true
  },
  signaller: signaller,
  room: room,
  ice: freeice(),
  localContainer: '#localVideo',
  remoteContainer: '#remoteVideo',
  plugins: [
    require('rtc-plugin-temasys')
  ],
  options: {}
});
```
This works flawless in webkit and firefox, but Internet explorer (the one you ironically need Temasys for ;) ) crashes with JS error: `Could not convert from struct FB::variant_detail::null to long`. 

The fun thing is that when I change the list of ice servers to an object containing an array as below, things do work:
```javascript
ice: {foo: freeice()},
```
I tried to find where this was coming from, but no luck yet. It also seems that none of the other browsers mind the extra layer there and are all working properly. And the name of the array does not seem to matter as well: `{stun: freeice()}`, `{test: freeice()}`, `{foo: freeice()}` all work fine.

**Comment by DamonOehlman** [comment]
Interesting - I'll have a look tomorrow as well and see if I can work out what is going on...

**Comment by nathanoehlman** [comment]
@Neograph734 This should be fixed now in 1.1.2. See the referenced PR for more details.

**Comment by Neograph734** [comment]
Awesome, thanks!
